onebusaway-android/src/main/java/org/onebusaway/android/ui/ArrivalsListFragment.java

```diff
@@ -1616,6 +1621,23 @@ private void addToDB(ObaStop stop) {
         ObaContract.Stops.insertOrUpdate(stop.getId(), values, true);
     }
 
+    private void addRouteToDB(ObaArrivalInfo info) {
```

Comment:
Could you please refactor this method to `QueryUtils`, and see if you can make RouteInfoListFragment and TripInfoActivity use the same method?  We have very similar code floating around in these methods (basically, most places where `ObaContract.Routes
                        .insertOrUpdate()` is used) and it would be nice to get them all using the same method.